js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -204,15 +209,14 @@ const OperatorSignInModalContent = ({
             submit(
               badge,
               radio,
-              filterExpired(certifications.result, now),
-              getMissing(certifications.result, "blue"),
+              certificationStatus,
               setComplete,
               setLoading,
               onComplete,
             );
           }}
           employees={employees.result}
-          certifications={certifications.result}
+          certificationStatus={getStatus(certifications.result, now, "blue")}
```

Comment:
Why recalculate this instead of using the variable `certificationStatus` from earlier?